mobify/platform-scaffold
# Use the new recommended method of hooking in the Redux devtools

This PR removes the deprecated method of hooking into the Redux dev tools in favour of the method currently recommended in their github readme. (https://github.com/zalmoxisus/redux-devtools-extension#12-advanced-store-setup)

The new setup also allows Immutable and plain JS objects to be distinguished in the dev tools.

 **JIRA**: N/A

## Changes
- Change the redux devtools integration

## How to test-drive this PR
- Preview the site
- Open the Redux dev tools
- See that they still operate correctly.
